ffw/UIRPC.js

```diff
@@ -876,11 +876,11 @@ FFW.UI = FFW.RPCObserver.create(
                   },
                   'numCustomPresetsAvailable': 10
                 },
-                'audioPassThruCapabilities': {
+                'audioPassThruCapabilities': [{
```

Comment:
```suggestion
                'audioPassThruCapabilitiesList': [{
```
`audioPassThruCapabilities` will always be processed as an object according to the `HMI_API`